src/wrappers/error-wrapper.js

```diff
@@ -36,6 +36,10 @@ export default class ErrorWrapper implements BaseWrapper {
     return false
   }
 
+  visible (): boolean {
+    return false
```

Comment:
This should throw an error, like the other methods.

Only exists returns false, everything else throws an error to indicate that find was not succesful

Reply:
Don't you think it should match the behavior of `exists()` method ? Having this return an error would make assertions on non-visible elements more verbose.

Reply:
No, I think it should throw an error. `exists` checks if find is successful, so it makes sense to return false. If find('div') is not successful, exists should return false.

Every other method checks something on the Wrapper element/ instance.

We need to make a distinction between a node being visible and a node being present.